src/redux/wallets/reducer.ts

```diff
@@ -38,7 +38,8 @@ const newBalance = (oldBalance: any, action: any) => ({
         : new BigNumber(oldBalance?.available).toFixed(),
     total: action.data.balance
         ? new BigNumber(action.data.balance.total).toFixed()
-        : new BigNumber(oldBalance?.total).toFixed()
+        : new BigNumber(oldBalance?.total).toFixed(),
+    detailed: action.data.balance ? action.data.balance.detailed || {} : oldBalance?.detailed || {}
```

Comment:
Safer to use:
`action.data.balance?.detailed || oldBalance?.detailed || {}`

Reply:
Fixed :)